chassis/core/src/main/java/com/griddynamics/jagger/util/generators/ConfigurationGenerator.java

```diff
@@ -37,25 +39,49 @@
     private MetricLogProcessor metricLogProcessor;
     private ProfilerLogProcessor profilerLogProcessor;
     private DurationLogProcessor durationLogProcessor;
-    private Map<String, JTestSuite> userJTestSuites;
+    private Map<String, JTestSuite> userJTestSuites = Collections.emptyMap();
+    private boolean useBuilders;
+    private final String jTestSuiteNameToExecute;
+    private Map<String, Configuration> configurations = Collections.emptyMap();
+    
+    public ConfigurationGenerator(String jTestSuiteNameToExecute) {
+        this.jTestSuiteNameToExecute = jTestSuiteNameToExecute;
+    }
     
     public Set<String> getUserJTestSuiteNames() {
-        if (userJTestSuites == null) {
-            return Collections.emptySet();
+        if (useBuilders) {
+            return new HashSet<>(userJTestSuites.keySet());
         }
-        return new HashSet<>(userJTestSuites.keySet());
+        return new HashSet<>(configurations.keySet());
     }
     
     @Autowired(required = false)
     public void setUserJTestSuites(List<JTestSuite> userJTestSuites) {
         this.userJTestSuites = userJTestSuites.stream().collect(Collectors.toMap(JTestSuite::getId, identity()));
     }
     
+    @Autowired(required = false)
+    public void setConfigurations(Map<String, Configuration> configurations) {
+        this.configurations = configurations;
+    }
+    
+    public Configuration generate() {
+        if (useBuilders) {
+            return generate(jTestSuiteNameToExecute);
+        }
+
+        Configuration configuration = configurations.get(jTestSuiteNameToExecute);
+        if (configuration == null) {
+            throw new IllegalArgumentException(String.format("No Jagger configuration with name %s", jTestSuiteNameToExecute));
+        }
+        return configuration;
+    }
+    
+    
     public Configuration generate(String userJTestSuiteName) {
```

Comment:
i mean if you get rid of it there are no way to trigger any other test except the one with the name you passed to the constructor